x-pack/plugin/apm-data/src/yamlRestTest/resources/rest-api-spec/test/10_rollover.yml

```diff
@@ -0,0 +1,54 @@
+---
+setup:
+  - do:
+      cluster.health:
+        wait_for_events: languid
```

Comment:
It is not clear to me why we are waiting here

Reply:
The intention here is to wait for the apm-data plugin to install the templates. Is that not necessary, or is there a better way?

Reply:
In general this increases the chances that the templates would be installed before the test runs, but it doesn't guarantee it. So, in this specific case, I do not think it adds anything for two reasons:

1. The only test in this suite disables the apm plugin.
2. You skip the test anyway because we cannot guarantee that the templates will be installed, so why wait?

Reply:
Fair. The real reason it's there is because I copied another test ;)
I'll remove the setup from this file.